### ISSUE TITLE
Low contrast ratio on "Dark Mode"

### ISSUE THREAD

**Issue by nijynot** [nyaadevs/nyaa#365]
Turn Dark Mode on, and the contrast ratio between odd rows (#363636) and link color (#247fcc) is 2.9:1.
The ratio is even worse for a:visited (#205c90) which gives 1.7:1.
A common minimum contrast ratio is 4.5:1. See: https://www.w3.org/TR/WCAG/#visual-audio-contrast

I can make a PR and try to fix the problem if there's interest.

**Comment by CounterPillow** [comment]
>I can make a PR and try to fix the problem if there's interest.

Please do! There's not any person focused on style things right now, so this would be nice to have.

**Comment by S-ed** [comment]
It's just my IMO, but `light on dark` contrast generally should be lower compared to `dark on light` to minimize eye strain.

**Comment by Yuubari** [comment]
Honestly, even if the contrast is low, this is almost unreadable (links on trusted background):

![image](https://user-images.githubusercontent.com/7750448/30963464-23f25092-a456-11e7-85b5-9e381c696d51.png)

**Comment by fgsfdss** [comment]
In my opinion there's no real need for the links to be blue, they can be in the same smoky white as the size and date (#afafaf) since the filename is already in its own distinct column anyway. Makes it perfectly readable and easy on the eyes, while best maintaining a consistent and matching style.

Reducing saturation and increasing lightness works wonders on the green and red for S/L as well. e.g. #9f9 & #f99 instead of standard red and green.

**Comment by S-ed** [comment]
@fgsfdss Yeah.
I and @nijynot showed examples in issue #28  
His theme has really nice contrast btw (but it's too much for me).

**Comment by ReimuHakurei** [comment]
I think this is a duplicate of #28.
